miden-proto/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "miden-proto"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+prost = { version = "0.11" }
+tonic = { version = "0.9" }
+
+[build-dependencies]
+tonic-build = { version = "0.9" }
+protox = { version = "0.4" }
+prost = { version = "0.11" }
+miette = { version = "5.9", features = ["fancy"] }
```

Comment:
minor nit: we usually alphabetize dependency lists.

Reply:
done